src/app/js/admin/fields/sagas/updateField.js

```diff
@@ -30,8 +30,8 @@ export function* handleUpdateField({ meta: { form } }) {
         yield put(updateFieldError(error));
     } else {
         yield put(updateFieldSuccess(response));
+        yield put(updateField(fieldData));
```

Comment:
updateField name seems wrong, we just uploaded the field successfully, and then we call updateField action?